1.6.3 fails to build: use of undeclared identifier 'bind2nd'

http://package19.nyi.freebsd.org/data/122amd64-default-build-as-user/566258/logs/votca-1.6.3.log

Same with 2021-rc.1-228-g1754257:

```
            bind2nd(multiplies<double>(), norm));
            ^~~~~~~
            boost::container::bind2nd
/usr/local/include/boost/container/detail/algorithm.hpp:81:24: note: 'boost::container::bind2nd' declared here
inline binder2nd<Func> bind2nd(const Func& func, const T& arg)
                       ^
/usr/ports/science/votca/work/votca-2021-rc.1-228-g1754257/csg/src/libcsg/map.cc:97:15: error: use of undeclared identifier 'bind2nd'; did you mean 'boost::container::bind2nd'?
              bind2nd(multiplies<double>(), norm));
              ^~~~~~~
              boost::container::bind2nd
/usr/local/include/boost/container/detail/algorithm.hpp:81:24: note: 'boost::container::bind2nd' declared here
inline binder2nd<Func> bind2nd(const Func& func, const T& arg)
                       ^
2 errors generated.
```

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -6,6 +6,7 @@ Version 2021-rc.2 (released XX.01.21)
 
 -  fix CI on rawhide (#642)
 -  add auto-cancel workflow to GitHub Actions (#645)
+-  namespace bind2nd to fix build on FreeBSD (#668)
 
 Version 2021-rc.1 (released 15.01.21)
 =====================================

--- a/src/libcsg/map.cc
+++ b/src/libcsg/map.cc
@@ -85,7 +85,7 @@ void Map_Sphere::Initialize(Molecule *in, Bead *out, Property *opts_bead,
   double norm = 1. / std::accumulate(weights.begin(), weights.end(), 0.);
 
   transform(weights.begin(), weights.end(), weights.begin(),
-            bind2nd(multiplies<double>(), norm));
+            std::bind2nd(multiplies<double>(), norm));
   // get the d vector if exists or initialize same as weights
   vector<double> d;
   if (_opts_map->exists("d")) {
@@ -94,7 +94,7 @@ void Map_Sphere::Initialize(Molecule *in, Bead *out, Property *opts_bead,
     // normalize d coefficients
     norm = 1. / std::accumulate(d.begin(), d.end(), 0.);
     transform(d.begin(), d.end(), d.begin(),
-              bind2nd(multiplies<double>(), norm));
+              std::bind2nd(multiplies<double>(), norm));
   } else {
     // initialize force-weights with weights
     d.resize(weights.size());

